### PR TITLE
fix https://github.com/zeebe-io/spring-zeebe/issues/66

### PR DESCRIPTION
**Pull request by HuangXiaoyu3331**
add "builder.numJobWorkerExecutionThreads(configurationProperties.getNumJobWorkerExecutionThreads()); "
code in ZeebeClientStarterAutoConfiguration.class

### CODE REVIEW

**Comment by salaboy** [comment]
@HuangXiaoyu3331 can you sign the CLA ?

**Comment by HuangXiaoyu3331** [comment]
I have signed CLA, but The Travis CI build could not complete. It seems that the JDK version is too low

**Comment by HuangXiaoyu3331** [comment]
> I have signed CLA, but The Travis CI build could not complete. It seems that the JDK version is too low

I got it

**Comment by spulci** [comment]
> I have signed CLA, but The Travis CI build could not complete. It seems that the JDK version is too low

In my PR #68 i fixed support for Java 11 as well as Travis issue during its build (updating both pom dependencies for Java 11 and travis.yml descriptor). If the maintainer will approve the PR we'll switch to Java 11 definitively. Btw Travis actually removed support for OracleJdk11 so I usesd OpenJdk11 on Travis.

**Comment by HuangXiaoyu3331** [comment]
> > I have signed CLA, but The Travis CI build could not complete. It seems that the JDK version is too low
> 
> In my PR #68 i fixed support for Java 11 as well as Travis issue during its build (updating both pom dependencies for Java 11 and travis.yml descriptor). If the maintainer will approve the PR we'll switch to Java 11 definitively. Btw Travis actually removed support for OracleJdk11 so I usesd OpenJdk11 on Travis.

ok, thanks

**Comment by salaboy** [comment]
@spulci is this change 
```
    builder.numJobWorkerExecutionThreads(configurationProperties.getNumJobWorkerExecutionThreads());

```

already covered in your PR?

**Comment by spulci** [comment]
> @spulci is this change
> 
> ```
>     builder.numJobWorkerExecutionThreads(configurationProperties.getNumJobWorkerExecutionThreads());
> ```
> 
> already covered in your PR?

No, i left my Pr intentionally without this change so to be covered when you will merge both of them. Merging this plus #68 will cover all the properties

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ language: java
 sudo: false # faster builds
 
 jdk:
-- oraclejdk8
```
Comment (salaboy) [inline .travis.yml]: @HuangXiaoyu3331 can you please rebase this PR from master? This change is not needed anymore

Comment (HuangXiaoyu3331) [reply .travis.yml]: I haved rebase it, please confirm whether it is correct

**Review by salaboy — APPROVED** [review]

**Pull request merged**